src/BizHawk.Client.EmuHawk/tools/Lua/Libraries/TAStudioLuaLibrary.cs

```diff
@@ -459,6 +459,75 @@ public string GetMarker(int frame)
 			return "";
 		}
 
+		[LuaMethodExample("local markerframe = tastudio.getmarkerabove(100).Frame")]
+		[LuaMethod("getmarkerabove", "returns a table of the marker at or above the given frame with fields Frame and Text")]
+		public LuaTable GetMarkerAbove(int frame)

```

Comment:
Not entirely sure about the name containign "above", especially when it could also return a marker for the frame specified instead of the marker "above" that frame. Perhaps `GetMarkerFor(int frame)` would communicate this better, but I don't feel strongly about this; if you think this is fine keep it.

Reply:
In TAS Editor this behaviour was just called getmarker, but in TAStudio that function name is already in use. I will leave the name as is.